stream_alert/athena_partition_refresh/main.py

```diff
@@ -0,0 +1,206 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+import json
+import logging
+import os
+
+from datetime import datetime
+
+import backoff
+import boto3
+
+logging.basicConfig()
+LEVEL = os.environ.get('LOGGER_LEVEL', 'INFO')
+LOGGER = logging.getLogger('StreamAlertAthena')
+LOGGER.setLevel(LEVEL.upper())
+
+ATHENA_CLIENT = None
+DATABASE_DEFAULT = 'default'
+
+
+def _load_config():
+    """Load the Athena Lambda configuration file
+
+    Returns:
+        [dict] Configuration settings by file, includes two keys:
+            lambda: All lambda function settings
+            global: StreamAlert global settings
+    """
+    config_files = ('lambda', 'global')
+    config = {}
+    for config_file in config_files:
+        with open('conf/{}.json'.format(config_file)) as config_fh:
+            try:
+                config[config_file] = json.load(config_fh)
+            except ValueError:
+                LOGGER.error('The \'%s\' file could not be loaded into json', config_file)
+                return
+
+    return config
+
+
+def _backoff_handler(details):
```

Comment:
I like the clarity of `_backoff_handler`.  The implementation can also change over time